src/components/Dashboard/index.jsx

```diff
@@ -158,7 +162,9 @@ export default class Dashboard extends Component {
           {withSidebar && (
             <Hidden mdUp>
               <IconButton
-                className={classes.link}
+                className={classNames(classes.link, {
+                  [classes.leftMargin]: !header,
```

Comment:
`classes.menuIconButton`.

Reply:
@helfi92  I have made the requested changes, please review it.

Reply:
@sonakshisaxena1 I don't see the new changes. Did you make sure to push your changes to this branch?